distribution/changelog.txt

```diff
@@ -3,6 +3,7 @@
 - Feature: [#10807] Add 2x and 4x zoom levels (currently limited to OpenGL).
 - Feature: [#12712] Add TCP / socket plugin APIs.
 - Feature: [#12840] Add Park.entranceFee to the plugin API.
+- Feature: [#12853] Render "Height Marks on Land" on all zoom levels when enabled.
```

Comment:
We would consider this a Change, not a Feature. However, if we change this PR to only affect the new 2x and 4x zoom levels, a changelog entry won't be needed.